docs/references/faq.rst

```diff
@@ -1,40 +1,35 @@
 
-	Is the information or data I add open access?
-		No, access to your analysis is always restricted to your collaboration and by the permissions you assign (see the section on :ref:`project-access` for more details). Nothing is open access on CERN Analysis Preservation (except for the projects own source code). It is designed to be a safe environment for CERN physicists to use from the very beginning of starting their analysis and at any given moment in its lifetime. It is also meant to increase this lifetime.
-		
-		If you are searching for a service providing open access data or you have some data to share, you may want to check out `http://opendata.cern.ch/ <CERN Open Data>`_.
+**Is the information or data I add openly accessible?**
+        No. You assign the access permissions to your analysis. Access can be restricted to your collaboration or according to your preferences (see the section on :ref:`project-access` for more details). Nothing is by default open access on CERN Analysis Preservation (except for the projects' own source code). It is designed to be a safe environment for CERN physicists to use from the very beginning of starting their analysis and at any given moment in its lifetime. 
+        
+If you are searching for a service providing open access data or you have some data to share, you may want to check out `http://opendata.cern.ch/ <CERN Open Data>`_.
 
-	With all this rerunning, cloning and reusing - where is my work? Are you trying to automate analyses?
-		No, we do not want nor can we do that. An analysis is an extremely complex research work which is precisely the reason why we care to preserve it. After all, if it were easy to automate the process of an analysis, why store it? What we are trying to do is to help you with repetitive tasks, help you find information you need and support your review and approval process so you can focus on the actual research.
+**I can edit, but can my collaborators edit my analysis too?**
+        This depends on what permissions you assign to your collaborators. All your collaboration colleagues should be able to read it (read-only access), but only those you invite specifically (by email or e-group) can edit as well.
 
-	I keep on hearing this word ... and I do not know what it means. Can you help me?
-		Sure, if you cannot find an explanation :doc:`here <glossary>` simply ask us :doc:`directly <../community/support>` and we will help you.
+**Are you trying to automate all analyses?**
+        No. Physics analyses are incredibly complex research work, which is precisely the reason why they are invaluable for preservation. What we are trying to do is help you with repetitive tasks, help you find the information you need, and support your review and approval process so that you can focus on the actual research.
 
-	
+**Do I have to use this rather long form or what other options do I have?**
```

Comment:
Please see the review comments I made in #739

Reply:
👍

Reply:
there are no comments in #739

Reply:
That was my bad, they were left pending. They're visible now.